doc/classes/DisplayServer.xml

```diff
@@ -1512,7 +1512,7 @@
 			Makes the mouse cursor hidden if it is visible.
 		</constant>
 		<constant name="MOUSE_MODE_CAPTURED" value="2" enum="MouseMode">
-			Captures the mouse. The mouse will be hidden and its position locked at the center of the screen.
+			Captures the mouse. The mouse will be hidden and its position locked at the center of the windowmanager window.
```

Comment:
"window manager's window", but it's not clear to me what's the difference between a "window manager's window" and a "window", and why the precision is useful. Can you have multiple WMs at the same time?

Reply:
There is a distinction between a OS-native window and an embedded Godot-Window.
Mouse-capture works only for OS-native windows.

I was unable to find a better name for this and I am open to suggestions.

It is possible to have multiple OS-native windows and also multiple embedded Godot-Windows at the same time.